### DISCUSSION TITLE
Can't get verified

### DISCUSSION THREAD

**Discussion by Tim0310ne** [community/community#102831] [GitHub Education]
### Hi everyone,

Your application cannot be reviewed until you fix the following:
Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your last name exactly as it appears in your academic affiliation document. If you have only a single legal name, enter it in both the first and last name fields.
Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.
This is what happens every time when I try to apply for GitHub education. The problem is I use my English name in the school system but when I use it for the application, it just tell me my name is wrong.

**Comment by phuccoder2000** [comment]
Hi @Tim0310ne 
If you are consistently encountering issues with your GitHub Education application due to discrepancies in your name, and you are required to use the exact names as they appear in your academic affiliation document
1. Contact GitHub Support:
- Visit the GitHub Support page: GitHub Support
- Log in with your GitHub account if prompted.
- Click on "Contact us" or look for a similar option to initiate a support request.
- Clearly explain the situation, stating that your academic affiliation document uses your legal name, while your school system uses your English name. Emphasize the need for assistance in resolving the discrepancy.
2. Use Legal Name if Necessary:
- If your academic affiliation document uses your legal name, consider using your legal name during the application process, even if it differs from your preferred English name. This is to ensure consistency with the documentation provided.
